### PR TITLE
feat(vwc-inline): add more inline size options

### PR DESCRIPTION
**Pull request by rinaok**
### All Submissions:
* [ ] I followed the guidelines in our [Contributing document](https://github.com/orgs/Vonage/teams/vivid-core/discussions/2)?
* [ ] My code follows the code style of this project
* [ ] I checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

* [ ] A11y tests
* [ ] Ui tests
* [ ] Unit and coverage
* [ ] Test for Aria [matching roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles)
* [ ] [Check the markup](https://validator.w3.org/)
* [ ] [Custom Control Accessible Development Checklist](https://w3c.github.io/using-aria/#checklist)
* [ ] Check unused/missing dependencies

## Checklist:

* [ ] API Changes are documented in code and exported accordingly ([JSDoc](https://jsdoc.app/about-getting-started.html))

### CODE REVIEW

**Comment by rinaok** [comment]
@yinonov Should we change the name to vwc-layout now? Or later?

**Comment by yinonov** [comment]
> @yinonov Should we change the name to vwc-layout now? Or later?

add another PR, just to have clear comparison for review

File: components/inline/readme.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# vwc-inline
+
+## Properties
+
+| Property        | Attribute        | Type                              | Default   | Description                                                                      |
+| --------------- | ---------------- | --------------------------------- | --------- | -------------------------------------------------------------------------------- |
+| `columnSpacing` | `column-spacing` | `"sm" \| "md"`                    | "md"      | Sets the initial preferred spacing of a column from predefined available options |
+| `columnBasis`   | `column-basis`   | `"sm" \| "md" \| "lg" \| "block"` | "sm"      | Sets the initial preferred measure of a column from predefined available options |
+| `template`      | `template`       | `"fill" \| "fit"`                 | undefined | Sets the initial preferred template from predefined available options            |
```
Comment (yinonov) [inline components/inline/readme.md:9]: what do you think of `Auto-Sizing`? is it a better name? template is too generic in the grid sense

File: components/inline/src/vwc-inline.scss [hunk 2]
```diff
@@ -13,49 +14,82 @@ $items-min-inline-sizes: (
...
 
 /* #region SPACING */
 $items-spacings: (
-	// ! hardcoded values should be defined and auto-generated to, and by, design tokens
-	'sm': 24px,
-	'md': 32px
+	
+	'xs': 16px,
+	'md': 24px,
+	'xl': 32px
```
Comment (yinonov) [inline components/inline/src/vwc-inline.scss:30]:
```suggestion
	xs: 16px,
	md: 24px,
	xl: 32px
```

File: components/inline/src/vwc-inline.scss [hunk 3]
```diff
@@ -13,49 +14,82 @@ $items-min-inline-sizes: (
...
+}
+/* #endregion */
+
+/* #region GUTTERS */
+$items-gutters: (
+	
+	'xs': 16px,
+	'md': 24px,
+	'xl': 32px
+);
```
Comment (yinonov) [inline components/inline/src/vwc-inline.scss:48]: same as `$items-spacings` so why not keeping a common var? it's local only

File: components/inline/src/vwc-inline.scss [hunk 4]
```diff
@@ -13,49 +14,82 @@ $items-min-inline-sizes: (
...
+:host([column-basis="block"i]) {
+	.layout {
+		grid-template-columns: 1fr;
+	}
 }
 
 :host(:not(template)),
 :host([template="fit"i]) {
-	#{$inline-template}: auto-fit;
+	.layout {
```
Comment (yinonov) [inline components/inline/src/vwc-inline.scss:86]: there's a lot of repetitive `.layout` selector.
in sass u can parent select
e.g.
```css
.layout {
  :host(:not(template)) &,
  :host([template="fit"i]) & {
    #{$inline-template}: auto-fit;
  }
  // ... and all other parent selectors
}
```

File: components/inline/test/inline.test.js [hunk 5]
```diff
@@ -29,32 +30,25 @@ describe('inline', () => {
...
-			await waitNextTask();
-			const { shadowRoot: { firstElementChild: slot } } = actualElement;
-			const assignedElements = slot.assignedElements();
-			const [childEl] = assignedElements;
-			await waitNextTask();
-			expect(childEl.clientWidth).to.equal(165);
+	describe('default values', () => {
+		it('should have the default values', async () => {
+			const COMPONENT_PROPERTIES = {
+				columnBasis: 'sm', columnSpacing: 'md', inlineGutters: 'md'
```
Comment (yinonov) [inline components/inline/test/inline.test.js:40]: are we checking properties attributes sync? maybe important especially when the names aren't identical

**Review by yinonov — CHANGES_REQUESTED** [review]

**Review by yinonov — APPROVED** [review]

**Pull request merged**